### ISSUE TITLE
make the focused reader display div scrollable?

### ISSUE THREAD

**Issue by shadoof** [dhowe/ReadersJS#158]
and @dhowe shouldn't the text in this div be selectable/copy-able so that human readers can capture generated text that they particularly like ... ?

**Comment by dhowe** [comment]
I"m not sure how that would work, as new entries are being added to the bottom?

**Comment by shadoof** [comment]
(I haven't checked the code but) are then DOM items at the top of the display simply deleted (I guess so)? Would there be a way to keep them and make the div scollable?

I definitely think that we should either 1) make the text in this display selectable or 2) allow users to spool it to a file somehow. (1) must be much easier, right?

**Comment by dhowe** [comment]
I agree that the text in this section (not the title) should be selectable. See https://github.com/dhowe/ReadersJS/issues/164

**Comment by cqx931** [comment]
In my original draft, the focused reader display div used to be scrollable. 
New entries are always added to the bottom and old entries won't be removed from the div.
It looks almost the same as what we have currently, except that there is a scrollbar on the side, which could be used to retrieve previous record.
In this case, the scrolling and updating are two separate systems that don't interfere with each other...

**Comment by dhowe** [comment]
so what happens when the user scrolls and a new entry appears at the bottom?

**Comment by cqx931** [comment]
In that version, the program doesn't control the scrolls and only updates new entries at the bottom.

**Comment by dhowe** [comment]
so the updates happen off screen?

**Comment by dhowe** [comment]
I think that if we want to have a scroll, then we need to have a timer, where if the user doesn't manually scroll for... say 5 seconds, then the interface jumps to the bottom and shows the auto-updating view we have now 

alternatively we could store the log of outputs since the last focus switch, and add a button that either saves it to a txt file, or opens it in a new tab (preferable)

**Comment by dhowe** [comment]
@shadoof ?

**Comment by shadoof** [comment]
@dhowe  I would be happy with the alternative above, especially if it makes coding more straightforward: add button to open a tab with log of output since last change of focus. (Whoever does this: Please ensure that the text of the log can be selected and copied without any html tagging)
